### PR TITLE
Testing

### LINKED ISSUE #57: [Bug] Impossible to login after restore
After restoring my instance, I was not able to login. It only worked one time, but I wasn't able to change anything.
Then I had the same issue as described here: https://github.com/kanboard/kanboard/issues/3422 (TL;DR: impossible to connect)

Then I compared both install and restore script. And it seems that the restore script lack some command to change some file access permissions.
I changed those permissions, it works :)

I'll PR to correct this issue.

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## Problem
- *Files permissions are not restored correctly*

## Solution
- *upgrade to 1.20, fix nginx issue, open jsonrpc for kandroid*
- *here I use the same config than the install script.*
- Fix #57 (tested with a manual fix, same commands)

## PR Status
Work finished. Package_check, basic tests and upgrade from last version OK.  
Could be reviewed and tested.

## Validation
---
*Minor decision*
- [x] **Upgrade previous version** : Maniack C
- [x] **Code review** : Maniack C from #55 and #58
- [x] **Approval (LGTM)** : Maniack C from #55 and #58
- [x] **Approval (LGTM)** : JimboJoe from #55 and #58
- **CI succeeded** : [Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/kanboard_ynh%20testing%20(Official)/)  
When the PR is mark as ready to merge, you have to wait for 3 days before really merge it.

### CODE REVIEW

**Comment by Jibec** [comment]
hi, why is this release "pending"?

File: manifest.json [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
+  "version": "1.2.0-0",
   "url": "https://kanboard.net/",
   "license": "AGPL-3.0",
   "requirements": {
 		"yunohost": ">= 2.7.0"
   },
   "maintainer": {
-    "name": "mbugeia",
-    "email": "maxime@max.privy.place"
+    "name": "jibec",
```
Comment (frju365) [inline manifest.json]: Why remove mbugeia ?

Comment (frju365) [reply manifest.json]: In fact, I know the previous discussion was transfered on dev.yunohost.org, but I'm still against removing previous maintainer... for me this PR is not ok.

Comment (Jibec) [reply manifest.json]: Well, because I rewrote the package and wish to assume the maintenance of the app

Comment (frju365) [reply manifest.json]: see here : https://github.com/YunoHost-Apps/umap_ynh/blob/master/manifest.json#L14
2 maintainers.

Comment (frju365) [reply manifest.json]: well, yes, I agree with you. You are the maintainer now, but you still use work of mbugeia ? so, I think you need to keep his name, at least somewhere in the README, or we will wait for a solution (through the issue).

Comment (Jibec) [reply manifest.json]: We already discussed this in another package, I don't really remember what was the result, I think the manifest were going to evolve.
I also remember we discussed this on the app xmpp room before making the change. We are talking about something 4 month old already...

Maybe @maniackcrudelis can tell us the good practice here, I don't want to obfuscate the existence or the work of previous maintainer, but don't see the point to let inactive point of contact.

Comment (maniackcrudelis) [reply manifest.json]: Exactly, we have already told about that matter in another issue (I don't remember which one though).
The final answer was to add a field in the manifest for previous maintainers.
A additionnal field will not bother YunoHost, and we can keep the current maintainer as the real person to contact in need of. But keep all the previous maintainer in the manifest and like that respect their previous work.

Comment (frju365) [reply manifest.json]: yes : this issue : https://github.com/YunoHost-Apps/kanboard_ynh/pull/55... but to be honest I'm not really convinced by what was said...

Comment (maniackcrudelis) [reply manifest.json]: Do you have another idea or a reason to share against that idea ?

**Comment by maniackcrudelis** [comment]
Huh, by the way this release is waiting because it's a testing PR which I've done just after merging the PR in testing. This PR is here only to remind us that there's a testing currently running.
Because a testing have to be tested as least one week, this PR can be reviewed now.

File: manifest.json (hunk 1 above)
Comment (maniackcrudelis) [reply manifest.json]: So, Jibec, can you add this entry in the manifest please ?

**Comment by Jibec** [comment]
done: https://github.com/YunoHost-Apps/kanboard_ynh/pull/62

**Comment by Jibec** [comment]
@frju365 now that the manifest issue is fixed, by any chance, do you have time to try the upgrade?

**Comment by frju365** [comment]
Euh... no. I don't have anything to test. My server is down and I will buy RPI to set up again what U lost. Really sorry, but right now, I can test nothing.

**Comment by maniackcrudelis** [comment]
Will be merged in 3 days

**Pull request merged**